Dockerfile

```diff
@@ -37,15 +37,18 @@ RUN export DATABASE_URL=mysql2://localhost/temp && \
     apt-get install -y mysql-server && \
     /etc/init.d/mysql start && \
     cp config/app_config.yml.SAMPLE config/app_config.yml && \
+    cp config/database.yml.MySQL_SAMPLE config/database.yml && \
     bundle exec rake db:setup assets:precompile && \
     rm -Rf config/app_config.yml tmp/* && \
     /etc/init.d/mysql stop && \
     rm -Rf /run/mysqld /tmp/* /var/tmp/* /var/lib/mysql /var/log/mysql* && \
     apt-get purge -y --auto-remove mysql-server && \
     rm -Rf /var/lib/apt/lists/* /var/cache/apt/*
 
-# Make relevant dirs writable for app user
+# Make relevant dirs and files writable for app user
 RUN mkdir -p tmp && \
+    chown nobody config/app_config.yml && \
```

Comment:
It's too bad we have to do that, it removes one security barrier. But I think it's ok.

Reply:
since it's only read during startup i see no problem